### ISSUE TITLE
Enable color themes to improve contrast for accessibility

### ISSUE THREAD

**Issue by SteveL-MSFT** [PowerShell/PowerShell#3611]
[ContrastAnalyser](https://www.paciellogroup.com/resources/contrastanalyser/) determines if the color contrast is sufficient to make text legible for accessibility

For example, the default error message colors on Windows, the red foreground on the black background is insufficient and should be a brighter red according to this tool.  Should be able to define custom colors for Progress, Error, Verbose, etc... and package it as a theme.

**Comment by chuanjiao10** [comment]
I came to discuss:

Set in .bashrc:
Export TERM = putty-256color
or
Export TERM = Eterm-256color
or
Export TERM = st-256color

Set in xshell:
Terminal   --->    terminal type --->  putty-256color

environment:
centos 7

Anyone try it, it works fine, it makes your console color beautiful.
It is recommended to set this to the powershell console default.
I mean: hard coding this setting in powershell is better. Do not let it inherit from BASH.

**Comment by Jaykul** [comment]
You can't set things like that in PowerShell, @chuanjiao10 -- those settings depend on your **terminal**.

**Comment by bitcrazed** [comment]
@SteveL-MSFT A thought on this: As you know, in FCU, we updated the default console palette to provide better contrast on modern displays. PLUS, we shipped a new tool to allow users to load color "schemes" if they prefer (esp. useful to those who're color-blind/sensitive).

If PowerShell was to fix its palette use to use color[0] as background, etc. rather than color[5], it'd be able to enjoy the palette improvements we've made ;)
